<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PRO-IND-507" name="Employee_Tests.Wrkday_ManagerActions_ChngEmpWrkLoc" tests="1" failures="0" timestamp="3 Jun 2017 04:14:29 GMT" time="4.785" errors="1">
  <testcase name="chgEmpWrkLoc" time="4.785" classname="Employee_Tests.Wrkday_ManagerActions_ChngEmpWrkLoc">
    <error type="java.lang.AssertionError" message="command:waitForElementPresent||description:Expected element is not found||Fail">
      <![CDATA[java.lang.AssertionError: command:waitForElementPresent||description:Expected element is not found||Fail
	at com.thoughtworks.selenium.SeleneseTestBase.fail(SeleneseTestBase.java:393)
	at com.arsin.ArsinSeleniumAPI.reportStepDtlsWithScreenshot(ArsinSeleniumAPI.java:2553)
	at com.arsin.ArsinSeleniumAPI.reportException(ArsinSeleniumAPI.java:1911)
	at com.arsin.ArsinSeleniumAPI.effecta(ArsinSeleniumAPI.java:1348)
	at Wrkday_POM.login.login_submit(login.java:12)
	at Employee_Tests.Wrkday_ManagerActions_ChngEmpWrkLoc.chgEmpWrkLoc(Wrkday_ManagerActions_ChngEmpWrkLoc.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- chgEmpWrkLoc -->
</testsuite> <!-- Employee_Tests.Wrkday_ManagerActions_ChngEmpWrkLoc -->
